client/my-sites/posts/post-controls/post-control.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export const PostControl = ( { control } ) =>
+	<li className={ classNames( { 'post-controls__disabled': control.disabled } ) } >
```

Comment:
It also occurred to me that post controls style would be better off in the `/post-controls` folder for consistency.